Makefile

```diff
@@ -1,5 +1,7 @@
 include common.mk
 
+export GOOS=linux
```

Comment:
Maybe `export GOOS ?= linux`
also this probably belongs in common.mk
Did you hit any issues with this ? we should be able to make the agent (and importantly the unit tests) on darwin as well as on linux

Reply:
This one is necessary for the go-lint to work, but yes putting it in common is enough for all.